tap_dynamodb/deserialize.py

```diff
@@ -58,23 +58,31 @@ def _apply_projection(self, record, breadcrumb, output):
                 breadcrumb_key = breadcrumb[0].split('[')[0]
                 index = int(breadcrumb[0].split('[')[1].split(']')[0])
                 if output.get(breadcrumb_key):
-                    output[breadcrumb_key].append(record[breadcrumb_key][index])
```

Comment:
Explain what this code will input in the Talend-Stitch

Reply:
Added comment.